pkg/config/error.go

```diff
@@ -40,6 +40,15 @@ func (e ErrEmpty) Error() string {
 	return fmt.Sprintf("%s: empty: %s", e.Name, e.Msg)
 }
 
+type ErrIncluded struct {
```

Comment:
i don't see this being used. what is happening? it's used in core?

if you need it, can you think of a more descriptive name? the code below KeyNotFoundError is pretty clear what it relates to. ErrIncluded is not intuitive to me

Reply:
Yep, it's used in core

Reply:
Can you expand the error message so that is clear to the users?
Does this really need a special type? Or could `ErrInvalid` be leveraged instead:
```go
return fmt.Sprintf("%s: invalid value (%v): %s", e.Name, e.Value, e.Msg)
```
There is also the in-between option of having a special constructor for this case, but still returning another type, like we do for duplicates:
https://github.com/smartcontractkit/chainlink-relay/blob/02b76df27fb4369ee017ce5512df1f8041f1afce/pkg/config/error.go#L20-L22

Reply:
I didn't want to use ErrInvalid because of `Value`:

If you don't specifiy a value, i.e.
`ErrInvalid{Name: name, Msg: "msg"}`

You get `'<nil>'` in the resulting error string. 

If you specify a value and it's nil, a panic is thrown. 

I suppose I could have hardcoded value to something like 'set', but this felt hacky, and it felt like there could be a category of TOML values that should be expected to be unset.

Reply:
Why don't you want to include the value?
What is the ideal error message for users? We can work backwards from that.

Reply:
> Why don't you want to include the value?
Because the value doesn't matter - its presence does